bitbots_vision/config/yoeo_visionparams_sim.yaml

```
@@ -0,0 +1,69 @@
+bitbots_vision:
+  ros__parameters:
+    # Defining the default vision parameters
```

```suggestion
    # Defining the simulation parameters for YOEO vision
```
Same for normal YOEO params